dart/utils/urdf/urdf_world_parser.cpp

```diff
@@ -51,16 +51,27 @@
 
 const bool debug = false;
 
-namespace urdf{
+namespace dart {
+namespace utils {
+namespace urdf_parsing {
+
+Entity::Entity(const urdf::Entity& urdfEntity)
+  : model(urdfEntity.model),
+    origin(urdfEntity.origin),
+    twist(urdfEntity.twist)
+{
+  // Do nothing
+}
 
 // Implemented in urdf_parser/src/pose.cpp, for some reason nobody thought of putting it in the header
-bool parsePose(Pose &pose, TiXmlElement* xml);
+//bool parsePose(urdf::Pose& pose, TiXmlElement* xml);
```

Comment:
Yes, I should actually do a second pass through the code and delete a bunch of old things that aren't being used anymore.